packages/govern-console/src/Providers/Wallet.tsx

```diff
@@ -4,30 +4,29 @@ import { UseWalletProvider, useWallet, Wallet } from 'use-wallet'
 import { useChainId } from './ChainId'
 import { getNetworkNode, getUseWalletConnectors } from '../lib/web3-utils'
 
-type WalletAugmentedContextProps = {
+export type WalletAugmentedData = {
   wallet: Wallet<any>
   ethers: any
-} | null
+}
 
-const WalletAugmentedContext = createContext<WalletAugmentedContextProps>(null)
+const WalletAugmentedContext = createContext<WalletAugmentedData | null>(null)
```

Comment:
For `WalletAugmentedData` and similar, I made it so that `null` is part of the context, but not the data type itself. This is so that the associated hooks (e.g. `useWalletAugmented()` here) can return the data without `null`, which makes it easier for consumers (since they don’t have to test for `null`).

Reply:
Oh I see why now! Much more ergonomic then. :)